src/main/java/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,31 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoNumbers {
+	private final List<Integer> lottoNumbers = new ArrayList<>();
+
+	public LottoNumbers(int... lottoNumbers) {
+		for (int lottoNumber : lottoNumbers) {
+			this.lottoNumbers.add(lottoNumber);
+		}
+	}
+
+	public LottoNumbers(List<Integer> lottoNumbers) {
+		for (Integer lottoNumber : lottoNumbers) {
+			this.lottoNumbers.add(lottoNumber);
+		}
+	}
```

Comment:
일급 컬렉션을 불변 객체와 같이 만들어보는 건 어떨까요 ?
Collection에서 제공하는 unmodifiableList를 활용하면 리스트에 대한 추가, 삭제를 막아주는 기능으로 일급 컬렉션도 좀 더 불변 객체에 가까워질 수 있습니다 :)

추가적으로 생성자를 사용할 때 주 생성자라는 개념이 있는데 참고해보시면 좋을 것 같아요 :)
https://jackjeong.tistory.com/148

```suggestion
    private final List<Integer> lottoNumbers;

    public LottoNumbers(int... lottoNumbers) {
        this(Arrays.stream(lottoNumbers)
                .boxed()
                .collect(Collectors.toList()));
    }

    public LottoNumbers(List<Integer> lottoNumbers) {
        this.lottoNumbers = lottoNumbers;
    }
```